bin/README.md

```diff
@@ -0,0 +1,16 @@
+## Running the web tool on transitionmonitor's docker-container
+
+Ensure your working directory is PACTA_analysis, then run one of these workflows:
```

Comment:
maybe a comment about the Docker image `2diirunner:1.3.0` being necessary?

Reply:
Thanks!